### PR TITLE
Update scorm packages

### PR DESCRIPTION
**Pull request by bfcoder**
This enables updating scorm packages

### CODE REVIEW

File: client/js/scorm/components/scorm/course.jsx [hunk 1]
```diff
@@ -105,19 +165,32 @@ export default class Course extends React.Component {
       );
     }
 
+    if (this.state.opened) {
+      settings = (<Settings
```
Comment (Cory-Christensen) [inline client/js/scorm/components/scorm/course.jsx]: Yes

File: client/js/scorm/components/scorm/course.jsx [hunk 2]
```diff
@@ -73,26 +103,56 @@ export default class Course extends React.Component {
...
     const isAssignment = !_.isUndefined(this.props.course.lms_assignment_id);
     const isGraded = this.props.course.is_graded === Course.ImportTypes.GRADED;
     const assignmentButtonProps = {
       canvasUrl: this.props.canvasUrl,
       courseId: this.props.courseId,
       lms_assignment_id: this.props.course.lms_assignment_id,
     };
+    const updateInput = (
+      <input
+        id="upload"
```
Comment (Cory-Christensen) [inline client/js/scorm/components/scorm/course.jsx]: I was planning on changing that. Will do.

File: app/controllers/api/scorm_courses_controller.rb [hunk 3]
```diff
@@ -18,7 +18,9 @@ def index
     courses = scorm_cloud_service.list_courses(
       filter: ".*_#{params[:lms_course_id]}",
     )
-    courses[:response] = scorm_cloud_service.sync_courses(courses[:response])
+    if courses[:status] != 400
```
Comment (jbasdf) [inline app/controllers/api/scorm_courses_controller.rb]: Might change to something like:
if [400, 500, 403].include?(courses[:status])

Comment (jbasdf) [inline app/controllers/api/scorm_courses_controller.rb]: Check to see what typical status codes come back from scorm engine

**Pull request merged**